bazel/api-golden/test/BUILD.bazel

```diff
@@ -1,4 +1,17 @@
 load("//bazel/api-golden:index.bzl", "api_golden_test", "api_golden_test_npm_package")
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin")
+
+package(default_visibility = ["//bazel/api-golden/test:__pkg__"])
+
+copy_to_bin(
+    name = "test_package_golden_dir",
+    srcs = glob(["goldens/test_package/**"]),
+)
+
+copy_to_bin(
+    name = "pkg_no_exports_field_golden_dir",
+    srcs = glob(["goldens/pkg_no_exports_field/**"]),
+)
```

Comment:
This is all necessary now because RBE does not support tree artifact source directories